lib/msal-common/test/client/RefreshTokenClient.spec.ts

```diff
@@ -139,27 +115,6 @@ describe("RefreshTokenClient unit tests", () => {
             });
         });
 
-        it("Checks whether performance telemetry startMeasurement method is called",async () => {
-            const spy = jest.spyOn(stubPerformanceClient, 'startMeasurement');
-            
-            const client = new RefreshTokenClient(config,stubPerformanceClient);
-            sinon.stub(RefreshTokenClient.prototype, <any>"executePostToTokenEndpoint").resolves(AUTHENTICATION_RESULT);
-            
-            await client.acquireToken(refreshTokenRequest);
-            expect(spy).toHaveBeenCalled();
-            spy.mockClear();
-        });
-
-        it("Checks whether performance telemetry endMeasurement method is called",async () => {
-            const spy = jest.spyOn(stubPerformanceClient, 'endMeasurement');
-            
-            const client = new RefreshTokenClient(config,stubPerformanceClient);
-            sinon.stub(RefreshTokenClient.prototype, <any>"executePostToTokenEndpoint").resolves(AUTHENTICATION_RESULT);
-            
-            await client.acquireToken(refreshTokenRequest);
-            expect(spy).toHaveBeenCalled();
-            spy.mockClear();
-        });
```

Comment:
Why are these tests deleted?

Reply:
Added back, had removed to make it consistent with earlier made changes (i.e Perf Client a part of Client Config), have undoed those.